### PR TITLE
feat: task run, actor call, deprecate top level call

### LINKED ISSUE #117: Add support to run actor task localy
On Apify platform you can easily run the actor task. But you can not do it locally using apify CLI.

### PR DESCRIPTION
**Pull request by vladfrangu**
Closes #117 
Also starts covering the slightest part of #554 

Still needs tests!

### CODE REVIEW

**Comment by B4nan** [comment]
I believe we want to keep the command reshuffling to v1 (that's what the PR is doing, right?)

**Comment by vladfrangu** [comment]
> I believe we want to keep the command reshuffling to v1 (that's what the PR is doing, right?)

`apify call` is remaining as a command, but it's printing a warning that it's deprecated and people should start using `actor call`. I _can_ revert that part if we really want to, but I don't think its needed

**Comment by B4nan** [comment]
please do, i would rather keep this for v1 where we will do similar changes on other places too. let's keep the implementation (running tasks locally), i would only revert the location (so keep `apify call` as the main documented and suggested API now)

File: src/commands/call.ts [hunk 1]
```diff
@@ -84,68 +57,37 @@ export class CallCommand extends ApifyCommand<typeof CallCommand> {
...
+            }
 
             return {
                 userFriendlyId: `${actor.username}/${actor.name}`,
                 id: actor.id,
             };
         }
 
-        // Try fetching Actor directly by name
+        // Try fetching actor directly by name
```
Comment (B4nan) [inline src/commands/call.ts:90]: revert and make sure we use `Actor` everywhere (https://github.com/apify/apify-core/issues/11666)

**Review by B4nan — APPROVED** [review]

**Pull request merged**